app/models/track.py

```diff
@@ -9,19 +9,21 @@ class Track(db.Model):
     description = db.Column(db.Text)
     track_image_url = db.Column(db.Text)
     color = db.Column(db.String, nullable=False)
+    font_color = db.Column(db.String)
```

Comment:
Define a property in the track model (in `app/models/track.py`). Similar to the already existing `serialize` property. Then you can just use it like a db field in the API Model definition.